lib/python/Screens/VirtualKeyBoard.py

```diff
@@ -506,7 +506,7 @@ def __init__(self, session, title=_("Virtual KeyBoard Text:"), text="", maxSize=
 			"last": (self.cursorLast, _("Move the text cursor to the last character")),
 			"backspace": (self.backSelected, _("Delete the character to the left of text cursor")),
 			"delete": (self.forwardSelected, _("Delete the character under the text cursor")),
-			"erase": (self.eraseAll, _("Delete the all the text")),
+			"erase": (self.eraseAll, _("Delete  all the text")),
```

Comment:
Looks like there're two spaces. Does gettext match this?
```suggestion
			"erase": (self.eraseAll, _("Delete all the text")),
```

Reply:
done